app.py

```diff
@@ -1,35 +1,185 @@
-from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+import os
+
+import requests
+from flask import Flask, render_template, request, redirect
+from requests.exceptions import HTTPError
+import dateutil.parser
 
 app = Flask(__name__, static_url_path='/static')
 app.config.from_object('flask_config.Config')
```

Comment:
This line, flask_config, and any mention of SECRET_KEY can actually be deleted. They were just used by `session` to encrypt the cookie.

Reply:
Removed